src/Adapter/Platform/AbstractPlatform.php

```diff
@@ -115,19 +111,18 @@ public function getQuoteValueSymbol()
     public function quoteValue($value)
     {
         trigger_error(
-            'Attempting to quote a value in '.get_class($this).
+            'Attempting to quote a value in ' . get_class($this) .
```

Comment:
Yes we can. Was not sure if you wanted it saved in variable or not. Made it into parameter. And code standard for dot at beginning of line does not seem to apply here, since no equal sign so I put it at the end.